api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/AbstractPatientCalculation.java

```diff
@@ -57,4 +59,12 @@ protected static CalculationResultMap calculate(
     return Context.getService(PatientCalculationService.class)
         .evaluate(cohort, calculation, calculationContext);
   }
+
+  protected HivMetadata getHivMetadata() {
+    return Context.getRegisteredComponents(HivMetadata.class).get(0);
```

Comment:
@nthfloor rejected this aTM, we reverted it from the unit test framework. please go ahead and update to match master

Reply:
this is no longer there after merge